pkg/api/meta/restmapper.go

```diff
@@ -335,3 +351,13 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
```

Comment:
It needs one.  @deads2k FYI

On Sep 24, 2015, at 5:07 PM, Janet Kuo <notifications@github.com> wrote:

In pkg/api/meta/restmapper.go
<https://github.com/kubernetes/kubernetes/pull/14461#discussion_r40373260>:

> @@ -335,3 +351,13 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
>  	}
>  	return nil, false
>  }
> +
> +// ResourceInGroup verifies if the group has the resource in it
> +func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
> +	for _, t := range m {

How do we find a t of the right group? RESTMapper doesn't have a group
field.

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/14461/files#r40373260>.